pyvista/plotting/plotter.py

```diff
@@ -5803,21 +5896,26 @@ def add_point_scalar_labels(
         if not is_pyvista_dataset(points):
             points, _ = _coerce_pointslike_arg(points, copy=False)
         if not isinstance(labels, (str, list)):
-            msg = 'labels must be a string name of the scalars array to use or list of scalars'
+            msg = 'labels must be a string name of the scalars array to use or list of scalars'  # type: ignore[unreachable]
             raise TypeError(msg)
         if fmt is None:
             fmt = self._theme.font.fmt
         if fmt is None:
-            fmt = '%.6e'
+            fmt = '%.6e'  # type: ignore[unreachable]
         if isinstance(points, np.ndarray):
             scalars = labels
         elif is_pyvista_dataset(points):
-            scalars = points.point_data[labels]
+            scalars = points.point_data[labels]  # type: ignore[assignment, index]
         phrase = f'{preamble} {fmt}'
         labels = [phrase % val for val in scalars]
         return self.add_point_labels(points, labels, **kwargs)
 
-    def add_points(self, points, style='points', **kwargs) -> Actor:
+    def add_points(
+        self,
+        points: NumpyArray[float] | pyvista.DataSet,
```

Comment:
All points (here and elsewhere in this PR) should technically allow a single vector or vector array (type should match `DataSet.points`)
```suggestion
        points: VectorLike[float] | MatrixLike[float] | pyvista.DataSet,
```